pkg/commands/config_verify.go

```diff
@@ -0,0 +1,175 @@
+package commands
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	hcversion "github.com/hashicorp/go-version"
+	"github.com/pelletier/go-toml/v2"
+	"github.com/santhosh-tekuri/jsonschema/v5"
+	_ "github.com/santhosh-tekuri/jsonschema/v5/httploader"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+	"gopkg.in/yaml.v3"
+
+	"github.com/golangci/golangci-lint/pkg/exitcodes"
+)
+
+func (c *configCommand) executeVerify(cmd *cobra.Command, _ []string) error {
+	usedConfigFile := c.getUsedConfig()
+	if usedConfigFile == "" {
+		c.log.Warnf("No config file detected")
+		os.Exit(exitcodes.NoConfigFileDetected)
+	}
+
+	schemaURL, err := getSchemaURL(cmd.Flags(), c.buildInfo)
+	if err != nil {
+		return fmt.Errorf("get JSON schema: %w", err)
+	}
+
+	err = validateConfiguration(schemaURL, usedConfigFile)
+	if err != nil {
+		var v *jsonschema.ValidationError
+		if !errors.As(err, &v) {
+			return fmt.Errorf("[%s] validate: %w", usedConfigFile, err)
+		}
+
+		detail := v.DetailedOutput()
+
+		printValidationDetail(cmd, &detail)
+	}
+
+	return nil
+}
+
+func printValidationDetail(cmd *cobra.Command, detail *jsonschema.Detailed) {
+	if detail.Error != "" {
+		cmd.PrintErrf("jsonschema: %s does not validate with %s: %s\n",
```

Comment:
The current golangci-lint output when the config is invalid:
```
❯ ./golangci-lint config verify --schema jsonschema/golangci.next.jsonschema.json
jsonschema: issues does not validate with /properties/issues/additionalProperties: additionalProperties 'exclude-paths' not allowed
jsonschema: run does not validate with /properties/run/additionalProperties: additionalProperties 'execute-timeout', 'skip-dirs' not allowed
```

Maybe we should improve it:
```
❯ ./golangci-lint config verify --schema jsonschema/golangci.next.jsonschema.json
The config is invalid:
jsonschema: "issues" does not validate with "/properties/issues/additionalProperties": additionalProperties 'exclude-paths' not allowed
jsonschema: "run" does not validate with "/properties/run/additionalProperties": additionalProperties 'execute-timeout', 'skip-dirs' not allowed
```
?